CHANGELOG.md

```diff
@@ -3,9 +3,12 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 # [v1.6.0](Unreleased)
 ### Notes
-- This release supports API2200 minimally where we can use OneView API2200 with this SDK. No new fields are added/deleted to support API2200.
+- This release supports API2200 minimally where we can use OneView API2200 with this SDK and ImageStreamer REST API version 2000 (I3S v5.40).  
+  This release adds variable.tf file to provide easy access for adding Appliance Credentials through environment variables or during runtime. 
+  No new fields are added/deleted to support API2200. 
 
 ### Oneview Features supported
+- Deployment Plans
```

Comment:
Can you move this to "Image Streamer features supported" section?

Reply:
done